aisynphys/synapse.py

```diff
@@ -0,0 +1,119 @@
+import numpy as np
+from collections import OrderedDict
+from .avg_response_fit import get_pair_avg_fits
+
+
+def generate_synapse_record(pair, db, session, notes_rec, syn='mono', max_ind_freq=50):
```

Comment:
Can you move this function back to the pipeline module code? The usual separation is that code purely related to analysis goes in `aisynphys/`, whereas code that deals with moving data around in the DB goes in `aisynphys/pipeline`.